docs/reference/data-streams/set-up-a-data-stream.asciidoc

```diff
@@ -4,48 +4,81 @@
 
 To set up a data stream, follow these steps:
 
-. <<configure-a-data-stream-ilm-policy>>.
-. <<create-a-data-stream-template>>.
-. <<create-a-data-stream>>.
-. <<secure-a-data-stream>>.
+* <<create-index-lifecycle-policy>>
+* <<create-component-templates>>
+* <<create-index-template>>
+* <<create-data-stream>>
+* <<secure-data-stream>>
 
-You can also <<convert-an-index-alias-to-a-data-stream,convert an index alias to
+You can also <<convert-index-alias-to-data-stream,convert an index alias to
 a data stream>>.
 
-[discrete]
-[[configure-a-data-stream-ilm-policy]]
-=== Optional: Configure an {ilm-init} lifecycle policy
+IMPORTANT: If you use {fleet} or {agent}, skip this tutorial. {fleet} and
+{agent} set up data streams for you. See {fleet-guide}/data-streams.html[Data
+streams] in the {fleet} Guide.
 
-While optional, we recommend you configure an <<set-up-lifecycle-policy,{ilm}
-({ilm-init}) policy>> to automate the management of your data stream's backing
-indices.
+[discrete]
+[[create-index-lifecycle-policy]]
+=== Step 1. Create an index lifecycle policy
```

Comment:
We lost all the context about this being optional. I'd prefer to keep at least a blurb about how this is not required to use data streams